api/src/main/java/org/openmrs/module/fgh/mpi/FhirUtils.java

```diff
@@ -672,6 +672,53 @@ private synchronized static void initializeCachesIfNecessary() {
 		}
 	}
 	
+	public static Map<String, Object> generateMessageHeader() {
+		Map<String, Object> messageHeader = new HashMap<String, Object>();
+		
+		messageHeader.put("fullUrl", "http://metadata.epts.e-saude.net/fhir/bundle");
```

Comment:
Do not put urls in code, use global properties or something

Reply:
Ok